chainer/utils/_collections.py

```diff
@@ -0,0 +1,31 @@
+import collections
+import weakref
+
+import six
+
+
+if six.PY3:
+    RefCountFriendlyOrderedDict = collections.OrderedDict
+else:
+    # Reference counting cannot free keys in old `collections.OrderedDict`,
+    # where a doubly linked list is used to maintain the order.
+    class RefCountFriendlyOrderedDict(object):
```

Comment:
How about simply naming this as `OrderedDict`? (because semantically they're equivalent.)

Reply:
Thanks.  Renamed it.